BunnyHopMaster/Assets/Scripts/Player/PlayerProgress.cs

```diff
@@ -37,6 +39,10 @@ void Update()
                 Time.timeScale = 0;
             }
         }
+        if (SceneManager.GetActiveScene() == SceneManager.GetSceneByName(lastScenename) && didWin)
+        {
+            SceneManager.LoadScene("Credits");
```

Comment:
The scene name should be stored in a constant at the top of the file to prevent errors in the future